public/js/types.js

```diff
@@ -7,7 +7,7 @@ const Source = t.struct({
   url: t.union([t.String, t.Nil]),
 
   // Internal for Firefox for now
-  actor: t.String
+  actor: t.maybe(t.String)
```

Comment:
If you look at the Firefox client I haven't actually translated id->actor anywhere yet, so it's still used for loading source text and probably other places. I will remove it today.